cypress/support/pages/account/pageAccount.js

```diff
@@ -5,6 +5,28 @@ class Account {
 	static #settingsButton = '[data-testid="settings"]';
 	static #email = '[data-testid="user_email"]';
 	static #emailReadOnly = '[data-testid="user_email_readonly"]';
+	static #inputBoxCurrentPasswordOnUserSettings =
+		'[data-testid="settings_password_current"]';
+	static #inputBoxEmailOnUserSettings = "input[id='email']";
+	static #submitButtonOnAccountSettings = '[data-testid="submit_new_password_btn"]';
+	static #successMessageOnAccountSettings = '[data-testid="notification"]';
+
+	enterCurrentPasswordOnUserSettingsPage() {
+		const currentPassword = Cypress.env("password");
+		cy.get(Account.#inputBoxCurrentPasswordOnUserSettings).type(currentPassword);
```

Comment:
hide password while typing as its showing in the logs

Reply:
done!